lib/inspec/input_registry.rb

```diff
@@ -152,7 +152,7 @@ def bind_inputs_from_runner_api(profile_name, input_hash)
           file: loc.path,
           line: loc.lineno
         )
-        find_or_register_input(input_name, profile_name, event: evt)
+        find_or_register_input(input_name.to_s, profile_name, event: evt)
```

Comment:
Why is this necessary? We seem to have a ton of `to_sym` and `to_s` littered all over the code and I can't tell if it is cargo-culting, or our lack of hygiene, or something else. Shouldn't this be the responsibility of `find_or_register_input` to accept a wide(r) range of inputs but deal with them correctly?

Reply:
Sure, I'll happily move the call within the find_or_register_input method.